eng/deployment/postgresql-kafka-opensearch/README.md

```diff
@@ -0,0 +1,76 @@
+# PostgreSQL -> Kafka -> OpenSearch
+
```

Comment:
Please add a note briefly describing why this exists (presumably for setting up a functional development environment).